DashboardApplication/Program.cs

```diff
@@ -0,0 +1,67 @@
+using Microsoft.Extensions.Options;
+using OrchardCore.Admin;
+using OrchardCore.Environment.Shell;
+using OrchardCore.Recipes;
+using OrchardCore.ResourceManagement.TagHelpers;
+
+var builder = WebApplication.CreateBuilder(args);
+
+builder.Services.AddOrchardCore()
+    .AddCommands()
+    .AddSecurity()
+    .AddMvc()
+    .AddIdGeneration()
+    .AddEmailAddressValidator()
+    .AddSetupFeatures("OrchardCore.Setup")
+    .AddDataAccess()
+    .AddDataStorage()
+    .AddBackgroundService()
+    .AddScripting()
+
+    .AddTheming()
+    //.AddLiquidViews()
+    .AddCaching()
+    .ConfigureServices(services => services
+        .AddRecipes())
+
+    // OrchardCoreBuilder is not available in OrchardCore.ResourceManagement as it has to
+    // remain independent from OrchardCore.
+    .ConfigureServices(s =>
+    {
+        s.AddResourceManagement();
+
+        s.AddTagHelpers<LinkTagHelper>();
+        s.AddTagHelpers<MetaTagHelper>();
+        s.AddTagHelpers<ResourcesTagHelper>();
+        s.AddTagHelpers<ScriptTagHelper>();
+        s.AddTagHelpers<StyleTagHelper>();
+    })
+    
+    // Fallback redirect to Admin dashboard
+    .Configure((app, routes, services) => {
+        var shellSettings = services.GetRequiredService<ShellSettings>();
+        var adminOptions = services.GetRequiredService<IOptions<AdminOptions>>();
+        routes.MapFallback("/", req =>
+        {
+            var redirectUrl = !String.IsNullOrEmpty(shellSettings.RequestUrlPrefix) ? $"/{shellSettings.RequestUrlPrefix}" : "";
+            redirectUrl += $"/{adminOptions.Value.AdminUrlPrefix}" ;
+            req.Response.Redirect(redirectUrl);
+            return Task.CompletedTask;
+        });
+    }, 10000);
+
+var app = builder.Build();
+
+if (!app.Environment.IsDevelopment())
+{
+    app.UseExceptionHandler("/Error");
+    // The default HSTS value is 30 days. You may want to change this for production scenarios, see https://aka.ms/aspnetcore-hsts.
+    app.UseHsts();
+}
+
+app.UseHttpsRedirection();
+app.UseStaticFiles();
+
```

Comment:
```suggestion
app.MapGet("/", async context =>
{
var moduleLink = $"{context.Request.Scheme}://{context.Request.Host}/WeatherForecast";
var module2Link = $"{context.Request.Scheme}://{context.Request.Host}/Module2/hello";
await context.Response.WriteAsync($"Hello from Dashboard application! " +
$"
Get the weather from Module1 at <a href="{moduleLink}">{moduleLink}" +
$"
or say hello to Module2 at <a href="{module2Link}">{module2Link}" +
$"
or check out the <a href="/admin">admin dashboard");
});
```

Reply:
I would suggest adding a controller/action here and return a view that should show these links. This would be a better sample since no one will actually write their code this way. People usually create controller/action

Reply:
I tried to add Controller/Views (Home/Index) and Razor Pages (Pages/About.cshtml) but the urls /Home/Index or /About are not responding. 
I don't know what I am missing.

I will merge it and we will see if we can improve it.